<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MessageBoxClass</class>
 <widget class="QMainWindow" name="MessageBoxClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MessageBox</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton_about">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>100</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>about</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_critical">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>100</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>critical</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_warning">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>180</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>warning</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_info">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>110</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>info</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_quest">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>180</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>quest</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_aboutqt">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>190</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>aboutqt</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="MessageBox.qrc"/>
 </resources>
 <connections/>
</ui>
